_data/tools.yml

```diff
@@ -719,6 +719,21 @@
       url: https://trino.io/docs/current/admin/jmx.html
     - urltext: JMX connector
       url: https://trino.io/docs/current/connector/jmx.html
+- name: jOOQ
+  anchor: jooq
+  category: add-on
+  logo: /assets/images/logos/jooq.png
+  logosmall: /assets/images/logos/jooq-small.png
+  description: |
+    Java Object Oriented Querying (jOOQ) generates Java code from your database
```

Comment:
jOOQ stands for jOOQ Object Oriented Querying

Reply:
Haha .. awesome .. another recursive acronym!